operator/config/manager/manager.yaml

```diff
@@ -117,4 +97,4 @@ kind: ConfigMap
 metadata:
   name: operator-config
 data:
-  kubesaw.namespace=workspaces-system
+  kubesaw.namespace: toolchain-host-operator
```

Comment:
The format is different from before, was it wrong before or is it now or are both correct?

Reply:
Both appear to be correct (at least in older versions of kustomize), I tried this out on a cluster from cluster-bot and tests passed.  I think this format is a little clearer as to what the intention is.

Reply:
Great!